websites/A/AnimeUltima/presence.ts

```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+  clientId: "852239305983262761"
+}),
+browsingTimestamp = Date.now(),
+presenceData: PresenceData = {
+  largeImageKey: "logo",
+  smallImageKey: "browsing"
+};
+
+function clearPresenceData() {
+  delete presenceData.smallImageKey;
+  delete presenceData.state;
+  delete presenceData.details;
+  delete presenceData.startTimestamp;
+}
+
+presence.on("UpdateData", () => {
+  clearPresenceData();
+  presenceData.startTimestamp = browsingTimestamp;
```

Comment:
delete it please... and use presenceData inside event and add `startTimestamp` to presenceData object